finger/models.py

```diff
@@ -12,28 +13,87 @@ def update_data(self, data):
         for entry in data:
             username = entry.get('användarnamn')
             if username and re.match('^[a-z]+$', username) and username != 'stacken':
+
+                phone_list = [entry.get('arbtelefon')] + \
+                             [entry.get('hemtelefon')]
```

Comment:
Is there any reason to do [foo] + [bar] rather than [foo, bar] here?

(This is the first place, the question regards all instances)

Reply:
No good reason, I will fix this.